xarray/core/dataset.py

```diff
@@ -10274,6 +10276,10 @@ def groupby(
         restore_coord_dims : bool, default: False
             If True, also restore the dimension order of multi-dimensional
             coordinates.
+        **groupers : Mapping of hashable to Grouper or Resampler
```

Comment:
dang it. nice catch! thanks.

Reply:
I have a fix but can't get typing to work.

Reply:
It's these ones right?
```
xarray/core/dataset.py:10321: error: Incompatible types in assignment (expression has type "Mapping[Hashable, Any]", variable has type "dict[str, Grouper]")  [assignment]
```
Isn't it similar to #5690? Try using `Any` instead of `Hashable`.

Reply:
Nice, thanks!